src/app/_components/PillButton/component.js

```diff
@@ -1,21 +1,21 @@
+import React from 'react';
 import PropTypes from 'prop-types';
-import { cn } from '@/app/utils/cn';
-import { Paragraph } from '../Typography';
+import { Paragraph } from '@components';
+import { cn } from '@utils/cn';
+import { buttonColorVariant, buttonType } from './style';
 
-export function PillButton({
-  children,
-  type = 'button',
-  className,
-  variant,
-  colorVariant,
-  icon,
-  disabled = false,
-  onClick,
-}) {
-  const buttonVariant = icon ? variant?.icon : variant?.regular;
-  const { regular, hover, focused, active, disabled: disabledState } = colorVariant || {};
+export function PillButton({ children, className, icon, variant, colorVariant, ...props }) {
+  const buttonVariant = icon ? buttonType[variant]?.icon : buttonType[variant]?.regular;
+  const buttonColor = buttonColorVariant?.[variant]?.[colorVariant];
```

Comment:
Why do you use ?. with buttonColorVariant if it's imported module, are you not sure it exists?

Reply:
Thanks for comment! Just forgot to remove. Fixed.